plugins/CoreHome/angularjs/multipairfield/multipairfield.controller.js

```diff
@@ -63,6 +79,19 @@
                         hasAny = false;
                     }
                 }
+
```

Comment:
Not sure the above `else if ($scope.field2) {` is still needed? Could maybe remove the feature that only a field2 is defined but no field1? If I understand the logic correct?

BTW could slightly make above block more readable by using eg 
![image](https://user-images.githubusercontent.com/273120/74108000-589f2580-4bda-11ea-825d-be6cdc144fc2.png)

Reply:
adjusted the code accordingly